components/bl-treeSelect-component/src/index.js

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState } from 'react';
+
+import primereact from './lib/primereact.min';
+
+const { cn } = BackendlessUI.CSSUtils;
+const { TreeSelect } = primereact.treeselect;
+
+export default function TreeSelectComponent({ component, eventHandlers }) {
+  const { display, classList, style, options, label, selectionMode, filterVisibility, chipsDisplay } = component;
+  const { metaKeySelection, scrollHeight, disabled, resetFilterOnHide, filterInputAutoFocus } = component;
+  const { filterPlaceholder, selectedOptionKey, emptyMessage } = component;
```

Comment:
You can also try to write like this
`const {`
         `display, classList, style, options, label, selectionMode, filterVisibility, chipsDisplay,`
        ` metaKeySelection, scrollHeight, disabled, resetFilterOnHide, filterInputAutoFocus,`
        ` filterPlaceholder, selectedOptionKey, emptyMessage`
`} = component;`

Reply:
From the experience of previous code reviews , if there are too many props, it is better to break them into a few logical lines.

Reply:
Does somebody argue that? Could you share a link to that? Because triple `const {...} = foo` looks strange to me.

Reply:
https://github.com/Backendless/ui-builder-library/pull/119#discussion_r1010460070

Reply:
https://github.com/Backendless/ui-builder-library/pull/119#discussion_r1010460070